ddl/util/syncer_test.go

```diff
@@ -178,7 +178,7 @@ func TestSyncerSimple(t *testing.T) {
 	if err != nil {
 		t.Fatalf("new session failed %v", err)
 	}
-	childCtx, cancel = context.WithTimeout(ctx, 100*time.Millisecond)
```

Comment:
Do we really need timeout in this test? Could we use `context.Background()`?

Reply:
I think we can